src/parser/core/modules/Checklist/Requirement.js

```diff
@@ -3,6 +3,7 @@ export default class Requirement {
 	_percent = null
 	value = null
 	target = 100
+	_weight = 1
```

Comment:
i don't _like_ it, but i don't really like checklist, either. will fix later with a nuke i guess.

Reply:
Seemed like the path of least resistance to make the bar (and thus the severity tiers) accurate for Overheal. I'd be fine with publicizing an unwritten rule that nothing new gets to use the weighting until we implement a replacement component.